### DISCUSSION TITLE
Bootstrap 6: what is the expected release time?

### DISCUSSION THREAD

**Discussion by mrholek** [twbs/bootstrap#41078] [General] [answered]
Hi Bootstrap Team,

First of all, I want to thank you for your excellent work and ongoing contributions to the developer community.

As the creator of [CoreUI](https://github.com/coreui/coreui), a UI library heavily relying on Bootstrap, I’m curious about the release plan for version 6. Could you share any updates on when we might expect the first alpha version?

I’m evaluating whether to wait for Bootstrap 6 or implement significant breaking changes on our end, such as dropping support for `node-sass`. Your guidance would help us plan our roadmap more effectively.

Looking forward to your response!

**Comment by julien-deramond** [comment, marked as answer]
Hi @mrholek 

>  I’m curious about the release plan for version 6. Could you share any updates on when we might expect the first alpha version?

The development for version 6 hasn’t started yet. Currently, we’re focused on wrapping up some remaining work for the 5.3.x releases, with a potential 5.4 release in the pipeline as well. So, it’s safe to say that version 6 won’t be arriving anytime soon.

> I’m evaluating whether to wait for Bootstrap 6 or implement significant breaking changes on our end, such as dropping support for `node-sass`.

Regarding the drop of support for `node-sass`, we haven’t made a final decision yet. It might happen with v5.4 or be deferred to v6. Since development for v6 hasn’t started, that's why we're evaluating if it could maybe be done in v5.4, even if it would be a breaking change for some...

**Reply by minisat0shi** [reply]
Hoping for a release of 5.4 soon!

**Comment by Andre-Rodrigues-Dev-Web** [comment]
Good morning, Bootstrap team!

Congratulations on all these years of working with the framework.

Regarding the new CSS features launched in 2024, is there the possibility of using them in any component?
